Samples/watchOS-Swift/watchOS-Swift WatchKit Extension/ContentView.swift

```diff
@@ -56,11 +59,36 @@ struct ContentView: View {
                 Button(action: captureTransaction) {
                     Text("Capture Transaction")
                 }
+                
+                Button(action: {
+                    viewModel.causeANR()
+                }) {
+                    Text(viewModel.anrText)
+                }
             }
         }
     }
 }
 
+class ContentViewModel: ObservableObject {
+    
+    @Published var anrText = "Cause ANR"
+    
+    func causeANR() {
+        
+        var i = 0
+        
+        for _ in 0...5_000_000 {
+            i += Int.random(in: 0...10)
+            i -= 1
+            
+            anrText = "\(i)"
+        }
```

Comment:
l: I would prefer a thread.sleep that is more predictable.

Reply:
I copied this logic from iOS-Swift. I didn't use Thread.sleep back then cause this is a more realistic use case of blocking the main thread. I would like to stick to it.